runtime/vm.go

```diff
@@ -1262,6 +1349,19 @@ mainloop:
 		}
 	}
 
+	if err != nil {
+		for v.ip < v.curIPLimit && v.framesIndex > 1 {
+			v.framesIndex--
+			v.curFrame = &v.frames[v.framesIndex-1]
+			v.curInsts = v.curFrame.fn.Instructions
+			v.curIPLimit = len(v.curInsts) - 1
+
+			filePos := v.fileSet.Position(v.curFrame.fn.SourceMap[v.curFrame.ip-1])
+			err = fmt.Errorf("%s\nin %s", err.Error(), filePos)
+		}
```

Comment:
I think this can be further reduced?
```golang
for v.framesIndex > 1 {
	v.framesIndex--
	v.curFrame = &v.frames[v.framesIndex-1]

	filePos := v.fileSet.Position(v.curFrame.fn.SourceMap[v.curFrame.ip-1])
	err = fmt.Errorf("%s\nin %s", err.Error(), filePos)
}
```

Reply:
Actually, now that I think about this, we're mutating the VM state partially, so we probably need to either
1. remove v.FrameInfo() because it may return partially correct info, or, 
2. mutate everything as if VM actually returned from all function calls

The main purpose of v.FrameInfo was to see where the error occurred during test. But it's not needed anymore because we have stack trace in error.

Reply:
Updated. 

True, I don't think v.FrameInfo adds any more useful information for debugging. Shall I remove it?

Reply:
Yes please

Reply:
Done